tests/containers/image.pm

```diff
@@ -28,8 +28,8 @@ sub run {
     my $versions = get_var('CONTAINER_IMAGE_VERSIONS', get_required_var('VERSION'));
     for my $version (split(/,/, $versions)) {
         my $images_to_test;
-        # Get list of images from CONTAINER_IMAGES_TO_TEST or use the default
-        unless ($images_to_test = get_urls_from_var('CONTAINER_IMAGES_TO_TEST')) {
+        # Get single image from CONTAINER_IMAGES_TO_TEST or array from get_suse_container_urls()
```

Comment:
i know that this is likely after @grisu48 request but i think it is good idea to point out that this is a ref to array.

Reply:
Done.